Frameworks/CoreText/CTRun.mm

```diff
@@ -117,7 +116,13 @@ void CTRunGetPositions(CTRunRef run, CFRange runRange, CGPoint* outPositions) {
         runRange.length = curRun->_range.length;
     }
 
-    memcpy(outPositions, &curRun->_glyphOrigins[runRange.location], sizeof(CGPoint) * runRange.length);
+    if (runRange.location != 0) {
+        UNIMPLEMENTED();
+        return;
+    }
+
+    outPositions->x = curRun->_xPos;
```

Comment:
DWrite only supports providing the points for the first glyph in the glyph run and not for every glyph in the run. We need so find a way to calculate this but the few things I tried did not work so need to get DWrite team's recommendation on what to do here.

I will add a TODO

---
In reply to: [77757825](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77757825) [](ancestors = 77757825)